src/Compilers/CSharp/Test/Semantic/Semantics/PropertyFieldKeywordTests.cs

```diff
@@ -1085,9 +1222,299 @@ .maxstack 8
 	}
 } // end of class C
 ");
+            Assert.Equal(1, accessorBindingData.NumberOfPerformedAccessorBinding);
+        }
+
+        [Fact]
+        public void TestAssigningFromConstructorNoAccessors()
+        {
+            var comp = CreateCompilation(@"
+public class C
+{
+    public C()
+    {
+        P = 5;
+    }
+
+    public int P { }
+
+    public static void Main()
```

Comment:
> public static void Main() [](http://example.com/codeflow?start=3&length=26)

It looks like the whole method can be deleted from the test scenario.

Reply:
Deleted. Thanks!